resources/views/templates/flash-message.blade.php

```diff
@@ -0,0 +1,30 @@
+@if ($message = Session::get('success'))
+    <div class="alert alert-success alert-dismissible fade show" role="alert">
+        <strong>{{ $message }}</strong>
+        <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+    </div>
+@endif
+
+@if ($message = Session::get('error'))
+    <div class="alert alert-danger alert-dismissible fade show" role="alert">
+        <strong>{{ $message }}</strong>
```

Comment:
can we refactor this file, I see only class is different based on session key, is there any way we can resuse the other piece of code.

Reply:
may be we can define in some array and match the class based on session key and the set it dynamically in class attribute